### DISCUSSION TITLE
Welcome to mruby Discussions!

### DISCUSSION THREAD

**Discussion by matz** [mruby/mruby#5219] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sasq64** [comment]
I am embedding MRuby in a C++ application with a graphical "playground".
I hope to create an environment where that allows for exploratory playing with programming, much like the 8-bit computers I grew up with.

**Reply by sasq64** [reply]
It is now testable online;
http://apone.org/toy
